tests/test_models.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+# Create your tests here.
+
+class ModelTests(TestCase):
+
+    def test_manufacturer_format_str(self):
+        manufacturer = Manufacturer.objects.create(name="Test",
+                                                   country="Test")
+
+        self.assertEqual(str(manufacturer), manufacturer.__str__())
```

Comment:
Ensure that the `__str__` method in the `Manufacturer` model is implemented to return the expected string format. The test currently checks if `str(manufacturer)` equals `manufacturer.__str__()`, which may not verify the actual string content.